src/rest/handlers/rest_utils.erl

```diff
@@ -0,0 +1,21 @@
+%%%-------------------------------------------------------------------
+%%% @author piotr
+%%% @copyright (C) 2021, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 07. Oct 2021 15:22
+%%%-------------------------------------------------------------------
+-module(rest_utils).
```

Comment:
the name of this module should be changed, as there isn't anything specific to REST or network in this module